examples/remote_rendering/src/server.rs

```diff
@@ -0,0 +1,59 @@
+use serde_diff::Diff;
```

Comment:
Better to put `server.rd` and `client.rs` into a `bin` folder, then users can run `cargo r -p remote_rendering --bin client`